desktop/extensions-sdk/dev/continuous-integration.md

```diff
@@ -0,0 +1,78 @@
+---
+title: "Continuous Integration"
+description: Automatically test and validate your extension.
+keywords: Docker, Extensions, sdk, CI, test, regression
+---
+
+In order to help validating your extension and ensure it's functional, Docker Desktop provides a set of tools to help you set continuous integration for your extension.
+
+## Setup CI environment with Github action
+
+You need Docker Desktop to be able to install and validate your extension.
+You can start Docker Desktop in Github Actions using the [Docker Desktop Action](https://github.com/docker/desktop-action), with the following step:
+
+```yaml
+steps:
+  - id: start_desktop
+    uses: docker/desktop-action/start@v0.1.0
+```
+
+> This action supports only Github Action Mac nodes at the moment ; you need to specify `runs-on: macOS-latest` for your end to end tests.
```

Comment:
In the GHA doc they call nodes runners: https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions#runners

```suggestion
> This action supports only Github Action macOS runners at the moment ; you need to specify `runs-on: macOS-latest` for your end to end tests.
```